src/nxt_runtime.c

```diff
@@ -789,6 +792,16 @@ nxt_runtime_conf_init(nxt_task_t *task, nxt_runtime_t *rt)
     rt->state = NXT_STATEDIR;
     rt->control = NXT_CONTROL_SOCK;
     rt->tmp = NXT_TMPDIR;
+    rt->conf_gen = 0;
+
+    time(&rawtime);
+    timeinfo = gmtime(&rawtime);  //convert to UTC
+    strftime((char*)buffer, 25, "%Y-%m-%dT%H:%M:%S.000Z", timeinfo);
+
+    rt->conf_ltime.length = strlen((char*)buffer);
+    rt->conf_ltime.start = nxt_malloc(rt->conf_ltime.length + 1);
```

Comment:
Thanks for changing the malloc(3), but now my next question is, where does this memory get free(3)'d?.

Reply:
I don't know when memory should be released. Can you explain me?

Reply:
We usually use nxt_mp_xxx api instead that you don't need to explicitly do memory free.
It will be released together with the mp destroy.